packages/core/src/Keymap.ts

```diff
@@ -40,13 +91,29 @@ export class Keymap {
         });
     }
 
+    /**
+     * Remove a shortcut.
+     *
+     * Removing a shortcut internally add another shortcut without `commandId`.
+     * The semantic of a shortcut without `commandId` is that it has been
+     * removed.  The reason is that if a lower priority define a shortuct, we
+     * need to have the information of a higher priority shortuct that contain
+     * the information of removal.
+     */
     removeShortuct(patternString: string, priority = 0): void {
         this._shortcutItems.push({
             pattern: this._getPatternFromString(patternString),
             priority,
         });
     }
 
+    /**
+     * Get all BoundCommand found when providing `keyEvent`.
+     *
+     * 1) Search through all shortucts
+     * 2) Filter to only get shortucts with hihest priority
+     * 3) Filter to only get shortucts that were not removed
+     */
```

Comment:
No params doc ? And as a general rule, it seems you like to describe the function inner workings in the docstring. I'm interested to know why. IMO, these three comment lines would be a lot more useful in the code of the function for these reasons:

- When I'm reading a function docstring, I am interested in what it does and how I can use it, not *how* it does it, I don't really care at that moment.
- When I'm reading the *code* of a function however, I don't want to go back and forth to the docstring trying to match the code I'm reading with one of the steps described up there. It allows you to put the comment at the exact point where it is relevant.